aiida_pseudo/cli/install.py

```diff
@@ -264,6 +260,6 @@ def cmd_install_pseudo_dojo(version, functional, relativistic, protocol, pseudo_
                 echo.echo_warning(msg)
 
         family.description = description
-        family.set_cutoffs(cutoffs, default_stringency=default_stringency)
+        family.set_cutoffs(cutoffs, default_stringency=default_stringency, unit='Eh')
```

Comment:
I suppose this is a typo?

```suggestion
        family.set_cutoffs(cutoffs, default_stringency=default_stringency, unit='Ev')
```

Reply:
Uhm, no? I think the cutoffs of the Pseudo Dojo tables are in Hartree, aren't they?

Reply:
Oh right 🤦  I forgot about `Eh` being an alternative for `Ha`. Seems `pint` also doesn't accept `Ha`.